lib/spack/env/cc

```diff
@@ -233,38 +233,44 @@ fi
 
 command="${0##*/}"
 comp="CC"
+vcheck_flags=""
 case "$command" in
     cpp)
         mode=cpp
         debug_flags="-g"
+        vcheck_flags="${SPACK_ALWAYS_CPPFLAGS}"
```

Comment:
I couldn't come up with a better name for these flags, please let me know if you have better ideas.